src/nonempty.rs

```diff
@@ -50,10 +49,10 @@ where
 
     #[inline]
     pub fn contains_rect(&self, rect: &Self) -> bool {
-        self.min_x() <= rect.min_x()
-            && rect.max_x() <= self.max_x()
-            && self.min_y() <= rect.min_y()
-            && rect.max_y() <= self.max_y()
+           self.min_x() <= rect.min_x()
+        && rect.max_x() <= self.max_x()
+        && self.min_y() <= rect.min_y()
+        && rect.max_y() <= self.max_y()
```

Comment:
Please revert the formatting change.

Reply:
May be reformat it as 
```rust
self.min_x() <= rect.min_x() &&
rect.max_x() <= self.max_x() &&
self.min_y() <= rect.min_y() &&
rect.max_y() <= self.max_y()
```
That style also appearing in the library and it is more readable

Reply:
Everyone has there own style preference, I personally don't think this is more readable and that can be bikeshedded all night long. Let's please just revert to the official rust style since sooner or later another rustfmt pass would change it again anyway.

Reply:
Or, reverted